library/accessory_workflows/build_bwa_reference/bwa-mk-index.wdl

```diff
@@ -0,0 +1,52 @@
+version 1.0
+
+workflow BuildBWARef {
+  input {
+      String ref_name
+      File reference_fasta
```

Comment:
Having just trudged through tech debt related to references, I'd suggest that you include a link with the location of the reference you use with this script and any expectations you're aware of that exist in the downstream pipeline (e.g. is it going to fail if these files are used from older reference versions?)